src/InodeCache.cpp

```diff
@@ -178,7 +199,11 @@ spin_lock(std::atomic<pid_t>& owner_pid, const pid_t self_pid)
         prev_pid = lock_pid;
         reset_timer = true;
       }
+#ifndef _WIN32
       sched_yield();
+#else
+      SwitchToThread();
+#endif
```

Comment:
You could just use [std::this_thread::yield](https://en.cppreference.com/w/cpp/thread/yield)